### PR TITLE
feat: switch styling & configuration card enhancement

### PR DESCRIPTION
**Pull request by haneabogdan**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by rangoo94 — APPROVED** [review]

File: src/styles/globalStyles.ts [hunk 1]
```diff
@@ -472,6 +472,22 @@ export const GlobalStyle = createGlobalStyle`
       fill: ${Colors.whitePure};
     }
   }
+
+  /* Switch */
+
+  .ant-switch {
+    background-color: ${Colors.slate600};
```
Comment (rangoo94) [inline src/styles/globalStyles.ts:479]: It's using `@disabled-color` in the internal implementation.

Maybe instead it should be either done by:
* replacing the `@disabled-color` to this one
* keeping the color as before
* replacing other styles as well (`:focus`'s `box-shadow` on that color)
?

File: src/styles/globalStyles.ts [hunk 2]
```diff
@@ -472,6 +472,22 @@ export const GlobalStyle = createGlobalStyle`
...
+  }
+
+  .ant-switch-checked {
+    background-color: ${Colors.indigo400};
```
Comment (rangoo94) [inline src/styles/globalStyles.ts:483]: Should be modified with `@switch-color` in LESS

File: src/styles/globalStyles.ts [hunk 3]
```diff
@@ -472,6 +472,22 @@ export const GlobalStyle = createGlobalStyle`
...
+  }
+
+  .ant-switch-handle {
+    &::before {
+      background-color: ${Colors.whitePure};
```
Comment (rangoo94) [inline src/styles/globalStyles.ts:488]: It's controlled by `@switch-bg` in LESS

File: src/antd-theme/my-antd-theme.less [hunk 4]
```diff
@@ -86,3 +86,7 @@
 @label-required-color: #f43f5e;
 // steps
 @steps-background: transparent;
+// switch
+@disabled-color: #475569;
```
Comment (rangoo94) [inline src/antd-theme/my-antd-theme.less:90]: Have you checked if that's working OK with all the UI?

**Review by rangoo94 — APPROVED** [review]

File: src/antd-theme/my-antd-theme.less (hunk 4 above)
Comment (haneabogdan) [reply src/antd-theme/my-antd-theme.less:90]: I agree this is a very generic variable. Checked code and also App. We already overwrote the once we where interested in (e.g. `@btn-disable-color`, `@input-disabled-color` etc). moves to the top of the file for better visibility

**Pull request merged**